manifests/cluster-authentication-operator_02_config.crd.yaml

```diff
@@ -5,10 +5,49 @@ metadata:
 spec:
   scope: Cluster
   group: operator.openshift.io
-  version: v1
   names:
     kind: Authentication
     plural: authentications
     singular: authentication
-  subresources:
-    status: {}
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    # Ensure we will be able to deserialize the object into the golang type
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          required: ["managementState", "logLevel", "operandSpecs"]
+          properties:
+            managementState:
+              type: string
+            logLevel:
+              type: string
+            operandSpecs:
+              type: array
+              items:
+                type: object
+                required: ["name", "operandContainerSpecs", "unsupportedResourcePatches"]
+                properties:
+                  name:
+                    type: string
+                  operandContainerSpecs:
+                    type: object
+                    required: ["name"]
+                    properties:
+                      name:
+                        type: string
```

Comment:
@enj in operandContainerSpecs, there's also 
`Resources *corev1.ResourceRequirements `json:"resources,omitempty"``  do I need to add that?